ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -36,6 +37,7 @@ component:
   guidevane: "Control vanes that meter refrigerant (on a centrifugal chiller) or air (on an AHU)."
   heater: "Component which provides heat to media."
   humidifier: "Component which humidifies."
+  lighting: "artificial light source devices or natural illumination by capturing daylight."
```

Comment:
this should be adjusted to `light` or similar if its going to be a component (i.e. a discrete, countable, instance of a thing);

if the intention is for this to be a descriptor, `lighting` is acceptable; it would need to be moved however.

Reply:
I agree with you to change to "light" or "lamp" so that more lighting device can use this component subfield. The widely usage of "light" or "lamp" component is greater then "lighting" as a descriptor.

Reply:
I've recommended to use "lamp" as a source of light, because light is a phenomenon, not a component.
See my review comments.

Reply:
Yes I agree with you